rustls/src/msgs/handshake.rs

```diff
@@ -225,11 +226,13 @@ impl ServerNamePayload {
     }
 
     fn read_hostname(r: &mut Reader) -> Result<Self, InvalidMessage> {
+        use pki_types::ServerName;
```

Comment:
Nit: I think this should probably just be imported at the top of the file?

Reply:
Unfortunately there's already a `ServerName` type in this file (matching the structure defined in RFC6066). I think the more local `use` is preferable to a global one that needs to be renamed?